README.md

```diff
@@ -81,6 +81,54 @@ Alternately, you can use the `ghw.WithChroot()` function like so:
 cpu, err := ghw.CPU(ghw.WithChroot("/host"))
 ```
 
+### Consuming snapshots
+
+You can make `ghw` read from snapshots (created with `ghw-snapshot`) using
+environment variables or programmatically.
+
+The environment variable `GHW_SNAPSHOT_PATH` let users specify a snapshot
+that `ghw` will automatically consume. All the needed chroot changes will be
+automatically performed. By default, the snapshot is unpacked on a temporary
+directory managed by `ghw`, and cleaned up when no longer needed, avoiding
+leftovers.
+
+The rest of the environment variables are relevant iff `GHW_SNAPSHOT_PATH` is given.
+`GHW_SNAPSHOT_ROOT` let users specify the directory
+on which the snapshot should be unpacked. This moves the ownership of that
+directory from `ghw` to users. `ghw` will still clean up the unpacked snapshot
+when no longer needed.
+
+`GHW_SNAPSHOT_PRESERVE` is relevant iff `GHW_SNAPSHOT_ROOT` is given.
```

Comment:
Whoops :) You replaced the `GHW_SNAPSHOT_EXCLUSIVE` with `GHW_SNAPSHOT_PRESERVE` :) It was actually `GHW_SNAPSHOT_KEEP` that I was suggesting to change to `GHW_SNAPSHOT_PRESERVE`.

Reply:
ah, my fault! fixed!